sonar-scanner-engine/src/main/java/org/sonar/batch/scan/measure/MeasureCache.java

```diff
@@ -64,8 +64,8 @@ public Measure byMetric(Resource r, String metricKey) {
   }
 
   @CheckForNull
-  public Measure byMetric(String resourceKey, String metricKey) {
-    return cache.get(resourceKey, metricKey);
+  public Measure byMetric(String componentKey, String metricKey) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "Measure"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)